### ISSUE TITLE
Is it working?

### ISSUE THREAD

**Issue by Tufis16** [jiaozi158/UnitySSGIURP#4]
I tested on Unity 2022.3.22 and added in global volume and urp setting but nothing changed. Also changed forward/forward+/deffered not working.

**Comment by jiaozi158** [comment]
Hi, do you have SSGI enabled in the volume override?
![SSGI_Volume_Override](https://github.com/user-attachments/assets/bb6dc284-6447-436a-94ee-2b7369fd8673)

**Comment by jiaozi158** [comment]
By the way, I suggest upgrading to a higher (patch) version of Unity Editor before installing this package, such as 2022.3.35f1.

There was a [bug](https://discussions.unity.com/t/urp-lacking-render-pass-injection-points/918754/8) in older versions of URP related to the render queue, and I don't have time to find out which version it was fixed in.

**Comment by Tufis16** [comment]
I will test your recommended version and after i will tell the results. Thanks.

**Comment by Tufis16** [comment]
Yeah i tested in clean Unity 2022.3.38 urp project and same. Can't see any diffrence. And yes everything attached and enabled.

**Comment by jiaozi158** [comment]
That's strange. Does the camera's active URP renderer have SSGI (renderer feature)?

Camera's renderer in use:
![Camera_Renderer](https://github.com/user-attachments/assets/6a24794f-e5f1-4d3f-9cec-e98c572b175c)
URP renderer asset:
![URP Renderer](https://github.com/user-attachments/assets/45d7563f-834a-4210-801a-29aa705547d1)

**Comment by Tufis16** [comment]
I tried but no.

**Comment by jiaozi158** [comment]
Could you provide a small project (ex. the empty project you created) so that I can reproduce the problem?

![Project Folder](https://github.com/user-attachments/assets/90ec9f16-e447-4f8e-af46-e7d572dc585d)
Thank you very much!

**Comment by Tufis16** [comment]
Still same, i read all your replies. I follow this steps and more:
1- Download this "https://github.com/jiaozi158/UnitySSGIURP.git" with github "or just downloaded with zip"
2- Drag and dropped to project
3- Added to renderer feature and enabled
4- Added to Global Volume and enabled
5- Checked rendering pipelines "from cameras or graphics settings" no changes anything.

This is project:
https://we.tl/t-zCMD0JKSms

**Comment by jiaozi158** [comment]
Hi, could you try again (the latest version) and see if the issue is resolved?

I downloaded the project you provided and found that it seems to be the same problem mentioned in [#5](https://github.com/jiaozi158/UnitySSGIURP/issues/5).

**Comment by Tufis16** [comment]
I think intensity is too low. Max value is 0.95. How can we make more saturated or high intensity?

**Comment by jiaozi158** [comment]
> How can we make more saturated or high intensity?

Hi, the "0.95" value controls the intensity of temporal accumulation. 

I think you're looking for the "Indirect Diffuse Lighting Multiplier".
![Multiplier](https://github.com/user-attachments/assets/8f42bacc-2437-4dfd-ac5a-563efec2a842)

**Comment by Tufis16** [comment]
Yes i mean this! but it shows after setting "All Visible" "Show All Addinational Setting Properties"

![Missing](https://github.com/user-attachments/assets/58df9f0c-b9d2-4b69-a0a3-831bc388d718)

![All visible](https://github.com/user-attachments/assets/b82ad5c3-c79e-483a-9678-0276dce28a42)

**Comment by jiaozi158** [comment]
Thanks for reminding!

I don't think it should be an additional property, so I'll change that in the next update.
